pkg/discovery/probe/node_probe.go

```diff
@@ -196,11 +208,26 @@ func (nodeProbe *NodeProbe) createCommoditySold(node *api.Node, nodePodsMap map[
 		}
 	}
 
-	// Use Kubernetes service UID as the key for cluster commodity
+	// Create a new VMPMAccessCommodity to indicate if the node is schedulable. If the Node is schedulable, then
+	// the node will sell it. Otherwise, it doesn't sell. On the consumer side, if a pod can be scheduled by
```

Comment:
I had the same question when reviewing it yesterday.  But from my understanding of the code, it seems if a pod is NOT schedulable, it is inactive and it will be marked "not monitored" - meaning it is not participating in the market at all (according to what I learned from Dongyi).  So looks like it's ok, though I agree that we should probably make this implication more explicit.